packages/app-project/src/shared/components/withRequireUser/withRequireUser.js

```diff
@@ -0,0 +1,49 @@
+import counterpart from 'counterpart'
+import { Box, Stack } from 'grommet'
+import { inject } from 'mobx-react'
+import { func, shape, string } from 'prop-types'
+import React, { Component } from 'react'
+
+import en from './locales/en'
+
+counterpart.registerTranslations('en', en)
+
+function storeMapper (stores) {
+  const { isLoggedIn } = stores.store.user
+  // return { isLoggedIn }
+  return { isLoggedIn: false }
```

Comment:
This line forces the YourStats component to remain un-logged-in - is this a WIP?

I tested the code by removing `return { isLoggedIn: false }` and re-enabled `return { isLoggedIn }`, and that seems to get the YourStats component working properly.

Reply:
WOOPS yeah that's totally left in from debug